Automattic/wp-calypso
# State: Modularize site connection subtree

#### Proposed Changes

This PR is one of many working on the modularizing state in Calypso. This one handles the site connection sites subtree. By splitting out the `sites` tree, we'll be able to load as little as possible initially.

For more details on state modularization, see the [modularized state documentation](https://github.com/Automattic/wp-calypso/blob/master/docs/modularized-state.md) and p4TIVU-9lM-p2.

#### Testing Instructions

* Go to `/plugins/:site` on an Atomic site and pick a plugin.
* Verify you can install the plugin properly through the primary install button (could have a "Purchase and activate", "Upgrade and activate" or "Install" label depending on whether the chosen plugin is a marketplace plugin, and the site's current plan).
* Verify all tests still pass.

#### Pre-merge Checklist
